includes/class.llms.ajax.handler.php

```diff
@@ -718,13 +718,14 @@ public static function select2_query_posts() {
 		}
 		$post_types = implode( ',', $post_types_array );
 
-		$post_statuses       = sanitize_text_field( $_REQUEST['post_statuses'] );
+		$post_statuses       = sanitize_text_field( llms_filter_input( INPUT_POST, 'post_statuses', FILTER_SANITIZE_STRING ) );
```

Comment:
`sanitize_text_field()` is somewhat redundant when using `filter_input` with sanitization filters

Reply:
yes you're right, sorry lost a bit of time because of some composer (update) issues I'm experiencing and missed that :D. 

I'm writing some tests and will fix the above too.

Reply:
fixed in a791556